<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LP398" failures="9" tests="11" name="Verify logo presence" time="16.466" errors="0" timestamp="06 Aug 2024 12:03:34 GMT">
  <testcase classname="com.sample.tests.PositiveTest" name="testPizzaType" time="1.851">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.PositiveTest.testPizzaType(PositiveTest.java:54)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testPizzaType -->
  <testcase classname="com.sample.tests.PositiveTest" name="testPizzaType" time="1.608">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.PositiveTest.testPizzaType(PositiveTest.java:54)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testPizzaType -->
  <testcase classname="com.sample.tests.PositiveTest" name="testPizzaType" time="1.448">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.PositiveTest.testPizzaType(PositiveTest.java:54)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testPizzaType -->
  <testcase classname="com.sample.tests.PositiveTest" name="testPizzaType" time="1.331">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.PositiveTest.testPizzaType(PositiveTest.java:54)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testPizzaType -->
  <testcase classname="com.sample.tests.PositiveTest" name="testPizzaType" time="1.35">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.PositiveTest.testPizzaType(PositiveTest.java:54)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- testPizzaType -->
  <testcase classname="com.sample.tests.NegativeTest" name="checkCashonPickupRadiobutton" time="0.193">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Cash on Pickup should not be Selected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Cash on Pickup should not be Selected expected [false] but found [true]
at com.sample.pages.PizzaPage.checkCashonPickup(PizzaPage.java:290)
at com.sample.tests.NegativeTest.checkCashonPickupRadiobutton(NegativeTest.java:81)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkCashonPickupRadiobutton -->
  <testcase classname="com.sample.tests.NegativeTest" name="checkCreditCardRadiobutton" time="0.149">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Credit Card should not be Selected expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Credit Card should not be Selected expected [false] but found [true]
at com.sample.pages.PizzaPage.checkCreditCard(PizzaPage.java:275)
at com.sample.tests.NegativeTest.checkCreditCardRadiobutton(NegativeTest.java:72)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkCreditCardRadiobutton -->
  <testcase classname="com.sample.tests.NegativeTest" name="checkMissingName" time="0.934"/>
  <testcase classname="com.sample.tests.NegativeTest" name="checkMissingPhoneNum" time="0.764">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Missing phone number] but found [Thank you for your order! TOTAL: 13.5 Large 10 Slices - 2 toppings]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Missing phone number] but found [Thank you for your order! TOTAL: 13.5 Large 10 Slices - 2 toppings]
at com.sample.tests.NegativeTest.checkMissingPhoneNum(NegativeTest.java:35)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkMissingPhoneNum -->
  <testcase classname="com.sample.tests.NegativeTest" name="checkResetbutton" time="1.331">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
	Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Topping 1 dropdown is not reset. expected [Choose a Toppings 1] but found [Mozzarella cheese],
Topping 2 dropdown is not reset. expected [Choose a Toppings 2] but found [Olives]
at com.sample.pages.PizzaPage.checkForm(PizzaPage.java:238)
at com.sample.tests.NegativeTest.checkResetbutton(NegativeTest.java:97)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1570)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkResetbutton -->
  <testcase classname="com.sample.tests.NegativeTest" name="checkmaxLength" time="0.394"/>
</testsuite> <!-- Verify logo presence -->
